src/SumoCoders/FrameworkUserBundle/Resources/config/controllers.yml

```diff
@@ -62,3 +62,13 @@ services:
       - "@translator"
       - "@router"
       - "sumocoders_frameworkuser_index_index"
+
+  sumo_coders.user.controller.delete:
```

Comment:
I would change `sumo_coders` to `sumocoders` in this controller name and also in the other ones. We kinda use `sumocoders` everywhere it seems a bit silly to change it in this file.

Reply:
What do you mean with "sumocoders" everywhere else, in all .yml (controllers, listeners, repositories, routing,...) files it's "sumo_coders"...

Reply:
even the folder with the bundles is called "SumoCoders" and not "Sumocoders"

Reply:
You are correct. It's in `src/SumoCoders/FrameworkUserBundle/Resources/config/routing.yml` that we are using `sumocoders` instead of `sumo_coders`.

Reply:
Confused now, it's multi_user_* there... https://github.com/sumocoders/FrameworkMultiUserBundle/blob/master/Resources/config/routing.yml

Reply:
@jonasdekeukelaere I mean this one: https://github.com/jonasdekeukelaere/Framework/blob/4acaff1b7bb4832997c60d701022f088ac17b682/src/SumoCoders/FrameworkUserBundle/Resources/config/routing.yml

Reply:
Fixed 034626b